docs/how-to/troubleshooting/troubleshoot-firing-alert-rules.md

```diff
@@ -0,0 +1,41 @@
+# Troubleshoot generic alert rules
+This guide contains steps towards troubleshooting firing generic alert rules. For detailed explanations on the design and goals of these rules, please refer to the [explanation page](/explanation/generic-rules).
+
+## How to troubleshoot the `HostDown` alert
+The `HostDown` alert is a sign that Prometheus is unable to scrape the metrics endpoint of the charm for whom this alert is firing. The methods below can help pinpoint the issue.
+
+### Ensure the workload is running
+It is possible that charm being scraped by Prometheus is not running. Use `juju ssh` to shell into the container containing the workload and ensure that the workload is running using `pebble services`. The Pebble service should be active. 
+
+### Ensure Prometheus is scraping the correct endpoint
+It is possible that Prometheus is not scraping the correct address, endpoint, or port. When a charm is related to Prometheus for scraping of metrics, the Prometheus config file appends the related charm's metrics endpoint address and port into its list of targets. For K8s charms, this address can be the pod's FQDN or the ingress address (if using Traefik for example). If the charm being scraped does not write the address correctly, then Prometheus will be unable to reach it.
+
+Another possibility is that the charm does not specify the correct port or endpoint for its metrics. When a charm instantiates the `MetricsEndpointProvider` object, it needs to set the correct port and metrics endpoint. For example, Alertmanager exposes its metrics at the `/metrics` endpoint on port 9093. Charm authors should ensure these values are correctly set, otherwise Prometheus may not have the correct information when attempting to scrape. Use the `ss` command to determine which ports are exposed by your workload.
```

Comment:
Just using this as another example - how you have "Use the `ss` command..." is fine here to keep it in-line because you're not supplying code that's meant to be run, but just describing what they could do (presumably the user knows this command)